custom/ilsgateway/views.py

```diff
@@ -113,8 +113,28 @@ class ILSConfigView(BaseConfigView):
     def page_context(self):
         context = super(ILSConfigView, self).page_context
         context['oneoff_tasks'] = OneOffTaskProgress.objects.filter(domain=self.domain)
+        ils_config = ILSGatewayConfig.for_domain(self.domain)
+        enabled = False
+        if ils_config:
+            enabled = ils_config.enabled
+        context['form'] = ILSConfigForm(initial={'enabled': enabled})
         return context
 
+    def post(self, request, *args, **kwargs):
+        ils_config = ILSGatewayConfig.for_domain(self.domain)
+        ils_config_form = ILSConfigForm(request.POST)
+        if not ils_config_form.is_valid():
+            return HttpResponseBadRequest()
```

Comment:
why not just return get() if the form is not valid? won't make much of a difference now but could make a difference if we add something to the form later